ovn_k8s/modes/overlay.py

```diff
@@ -217,6 +237,16 @@ def delete_logical_port(self, event):
 
         vlog.info("deleted logical port %s" % logical_port)
 
+        if delete_network_policies:
```

Comment:
it does not indeed since the pod at this stage is already deleted. I'll see if removing ACLs first simplifies the logic